### DISCUSSION TITLE
Welcome to FoxServer Discussions!

### DISCUSSION THREAD

**Discussion by Irwin1985** [VFPLegacy/FoxServer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sixtored** [comment]
Excelente oportunidad tenermos los foxeros al poder utilizar Foxserver pensemos que es ahora tenemos la posibilidad de expandir aun mas la vida a nuestros sistemas incluso poder expandir y brindar soluciones que antes nos parecian imposible realizarlas.-
Saludos.- I

**Reply by Irwin1985** [reply]
Hola Sixtored, me alegra que seas uno de los primeros que creyó en el proyecto y tan es así que ya tienes tu primera API en producción y en tiempo record. Felicidades y a seguir adelante...!
